### PR TITLE
residualvm: enable sndio by default

### PR DESCRIPTION
**Pull request by Kistelini**

### CODE REVIEW

**Comment by maxice8** [comment]
@newbluemoon

**Comment by maxice8** [comment]
On Tue, Jan 29, 2019 at 11:27:15AM -0800, newbluemoon wrote:
> newbluemoon commented on this pull request.
> 
> 
> 
> >  		 -e "s;_strip=strip;_strip=${XBPS_CROSS_TRIPLET}-strip;" \
>  		 -e "s;_ar=\"ar cru\";_ar=\"${XBPS_CROSS_TRIPLET}-ar cru\";" \
>  		 -e "s;_as=\"as\";_as=\"${XBPS_CROSS_TRIPLET}-as\";" \
>  		 -e 's;--prefix="$_sdlpath";;' \
>  		 -e 's;--prefix="$_freetypepath";;' \
>  		 -e "s;^_host_cpu=\"\";_host_cpu=\"${XBPS_TARGET_MACHINE}\";" \
>  		 -e 's;^_host_os="";_host_os="linux";' \
> -		 configure
> +		 -i configure
> 
> I like to have the `-i` right after the `sed` or `vsed`, it feels better for me this way. ;) Else no objections.
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/7926#pullrequestreview-197740354

with vsed then -i must be followed by the file

**Comment by newbluemoon** [comment]
@maxice8 Ah, ok then. :) Haven’t had the chance to follow up on `vsed`, yet.

File: srcpkgs/residualvm/template [hunk 1]
```diff
@@ -17,22 +17,23 @@ distfiles="http://www.residualvm.org/downloads/release/${version}/residualvm-${v
 checksum=f50c83bbc55a8121eefc279e83982b6ec590e608e145b7f750006619dd0bf9e9
 
 build_options="sndio"
+build_options_default="sndio"
```
Comment (newbluemoon) [inline srcpkgs/residualvm/template]: As this is the only build option, maybe remove the option altogether and just enable it?

Comment (Kistelini) [reply srcpkgs/residualvm/template]: All other packages have sndio as an option, too. It's enabled by default but it's still possible to get an sndio free system.

Comment (newbluemoon) [reply srcpkgs/residualvm/template]: I’m ok with either way. :)

**Pull request merged**